symantec_edr/docs/README.md

```diff
@@ -0,0 +1,213 @@
+# CTE Symantec EDR v1.0.0 Readme
+
+## Release Notes
+### 1.0.0
+#### Added
+
+- Initial Release.
+- Symantec EDR plugin (On Prem) fetches IoCs of the type of URL (Domain, IPv4, IPv6), and SHA256 from the Symantec EDR Entities Endpoint. This plugin does not support sharing of indicators.
+
+## Description
+
+The Symantec EDR (On Prem) plugin fetches IoCs of the type of URL (Domain, IPv4, IPv6), and SHA256 from the Symantec EDR platform. This plugin does not support sharing of indicators.
+
+## Plugin Scope
+
+| Type of data supported        |    |                                     
+|-------------------------------|------------------------------------------------------|
+| Fetched indicator types       | URL (Domains, IPv4 and IPv6), SHA256  |
+| Shared indicator types        |Not Supported   |
+
+## Prerequisites
+
+- Netskope Tenant (or multiple, for example, production and development/test instances)
+- Netskope Cloud Exchange: Threat Exchange Module
+- Netskope CTE Plugin
+- Connectivity to Symantec EDR platform
+
+## Connectivity to the following hosts
+
+- Connectivity to Symantec EDR instance.
+
+## Mappings
+
+### Pull Mapping
+
+|Netskope CE Fields|Symantec EDR Fields |
+|--|--|
+|value  | sha2, data_source_url_domain, device_ip|
+|firstSeen  | first_seen|
+|lastSeen  | last_seen|
+|severity|disposition, file_health|
+
+**NOTE:** Severity is mapped with both disposition and file_health, as we receive disposition while fetching the URLs and file_health while fetching the hash values.
+
+### Severity - Disposition Mapping
+
+|Netskope CE Severity|Symantec EDR Disposition |
+|--|--|
+|Unknown  |1 = unknown|
+|Low  | 0 = healthy/good|
+|High  | 2 = suspicious|
+|Critical|3 = bad|
+
+### Severity - file_health Mapping
+
+|Netskope CE Severity|Symantec EDR file_health |
+|--|--|
+|Unknown  | 2 = unknown |
+|Low  | 1 = good|
+|High  | 3 = suspicious|
+|Critical|4 = bad|
+|Medium|6 = leaning good|
+
+## API Details
+
+### List of APIs used
+
+|API Endpoint  |Method  | Use case  |
+|--|--|-- |
+| /atpapi/oauth2/tokens| POST|To get bearer token|
+|/atpapi/v2/entities |POST  | Pull entities from Symantec EDR platform|
+
+### Get Bearer Token
+
+**Endpoint:** /atpapi/oauth2/tokens
+
+**Method:** POST 
+
+**Content-Type:** application/x-www-form-urlencoded
+
+**Accept:** application/json
+
+**Authorization:** Bearer {token}
+
+#### Request Body
+```
+{
+    "client_id": "5106b812*****5c1cb53b38f4862b",
+    "client_secret": "9N8yf8aAZ8ZOMQ0NtUO-gb3KYWvBukUCIm&*********",

```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical
<details>
<summary><strong>Description</strong></summary>

A generic secret or password is an authentication token used to access a computer or application and is assigned to a password variable.

</details>


Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 61d16c33-4e18-4575-8cb6-c5b8b5a92bf6 -->